### PR TITLE
UI fixes for AAD+Organizations bug bash

### PR DESCRIPTION
**Pull request by chenriksson**
Fixes the following issues:
- #5379 : Cancel buttons on Manage Organization page
- #5379 : Account drop-down on Manage Organizations page
- Expander icons for Account, Manage Organization pages (regression from partials)
- Breadcrumb fix, per @anangaur

### CODE REVIEW

File: src/NuGetGallery/Scripts/gallery/page-manage-organization.js [hunk 1]
```diff
@@ -175,6 +201,11 @@
             };
         }
 
+        // Set up the section expanders.
+        for (var i in sections) {
+            configureSection(sections[i]);
+        }
```
Comment (chenriksson) [inline src/NuGetGallery/Scripts/gallery/page-manage-organization.js]: Code taken from `page-account.js`, also needed here to close sections on clicking "Cancel"

File: src/NuGetGallery/Views/Organizations/ManageOrganization.cshtml [hunk 2]
```diff
@@ -12,9 +14,9 @@
     <div class="row">
         <div class="col-md-12">
             <h1>
-                <a href="@Url.ManageMyOrganizations()">Manage Organizations</a>
+                <a href="@Url.ManageMyOrganizations()">Organizations</a>
```
Comment (chenriksson) [inline src/NuGetGallery/Views/Organizations/ManageOrganization.cshtml]: Breadcrumb fix

File: src/NuGetGallery/Views/Organizations/ManageOrganization.cshtml [hunk 3]
```diff
@@ -3,6 +3,8 @@
 @{
     ViewBag.Title = "Manage Organization";
     Layout = "~/Views/Shared/Gallery/Layout.cshtml";
+
+    TempData["Parent"] = this;
```
Comment (chenriksson) [inline src/NuGetGallery/Views/Organizations/ManageOrganization.cshtml]: "Parent" changes fix toggling of expander icons

File: src/NuGetGallery/Views/Users/Transform.cshtml [hunk 4]
```diff
@@ -75,5 +75,4 @@
             })
         });
     </script>
-    @Scripts.Render("~/Scripts/gallery/site.min.js")
```
Comment (chenriksson) [inline src/NuGetGallery/Views/Users/Transform.cshtml]: Fixes account drop-down issue on Transform page (script already included by Layout)

File: src/NuGetGallery/Views/Organizations/ManageOrganization.cshtml (hunk 3 above)
Comment (scottbommarito) [inline src/NuGetGallery/Views/Organizations/ManageOrganization.cshtml]: Maybe a more descriptive name? `TempData` could store anything. `ViewParent` perhaps?

Also can we use a constant for this?

**Comment by scottbommarito** [comment]
Note:  you assigned everyone instead of marking them as reviewers

**Pull request merged**